security.md

```diff
@@ -149,6 +140,16 @@ Using git would allow you to add outsource/freelance developers for a limited ti
 
 * There are extra protection products on top of an antivirus called EDR (Cyberreason, BlackCobalt) but these are usually costly.  
 
+### Hash your customer's passwords with a proper password hashing function
+
+
+If you can afford it, use a [third party authentication service](#customer-users-management) to handle password storage, password management and password recovery. 

```

Comment:
I don't agree that a *third party authentication service* is "security 101". This is a decision that needs to be weighed more carefully than **just hashing passwords right**.

Additionally, chances are that devs are going to pull a package and store internally in 90% of cases in the first instance/version/mvp. This "101" advice just reminds/informs them to use a hashing function specifically designed for passwords. I think adding the complexity of a third party service is more cognitive load and potentially another instance of [indirection](https://github.com/forter/security-101-for-saas-startups/pull/26#issuecomment-284969446). 

Aside: if you care for advice, try to keep the #UX of even this document in mind: you don't want it to become too big to navigate. Folks will churn if they can't move from section to section easily without being bogged down with so many different suggestions. You don't need to please everybody, [cater for all vendors](https://github.com/forter/security-101-for-saas-startups/issues/30) or cover every possible alternative. An opinionated view might help to keep this concise and easy to consume. HTH